.circleci/config.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       # otherwise.
       ALLOWED_FAILURE: "no"
       # Tell Hypothesis which configuration we want it to use.
-      MAGIC_FOLDER_HYPOTHESIS_PROFILE: "ci"
+      MAGIC_FOLDER_HYPOTHESIS_PROFILE: "magic-folder-ci"
```

Comment:
The profile is renamed so it doesn't conflict with the one defined by Tahoe-LAFS.